src/components/Modals/Modal.vue

```diff
@@ -231,7 +231,7 @@ export default {
   width: 15.5rem;
 }
 
-@media only screen and (max-width: 878px) {
+@media only screen and (max-width: 600px) {
```

Comment:
Same here.

Reply:
At 600px, we're starting to control for more mobile sized screens so we want the modal to have width 100% in that case.